README.md

```diff
@@ -2,18 +2,18 @@
 
 Rekór - Greek for “Record”
 
-Rekor's goals are to provide an immutable tamper resistant ledger of metadata generated within a software projects supply chain.  Rekor will enable software maintainers and build systems to record signed metadata to an immutable record. Other parties can then query said metadata to enable them to make informed decisions on trust and non-repudiation of an object's life-cycle. For more details visit the [rekor website](https://rekor.dev)
+Rekor's goals are to provide an immutable tamper resistant ledger of metadata generated within a software projects supply chain.  Rekor will enable software maintainers and build systems to record signed metadata to an immutable record. Other parties can then query said metadata to enable them to make informed decisions on trust and non-repudiation of an object's life-cycle. For more details visit the [rekor website](https://sigstore.dev)
```

Comment:
```suggestion
sigstore's goals are to provide an immutable tamper resistant ledger of metadata generated within a software projects supply chain. sigstore will enable software maintainers and build systems to record signed metadata to an immutable record. Other parties can then query said metadata to enable them to make informed decisions on trust and non-repudiation of an object's lifecycle. For more details visit the [sigstore website](https://sigstore.dev)
```

Reply:
Ah good point, so must of those will remain as rekor (as they refer to this project, but the last one is correct and should be changed; `[sigstore website](https://sigstore.dev)`